doc/src/manual/gun.asciidoc

```diff
@@ -257,6 +258,16 @@ keepalive (infinity)::
 
 Time between pings in milliseconds.
 
+keepalive_tolerance::
+
+The number of unacknowledged pings that can be tolerated before the connection
+is forcefully closed.
+When a keepalive ping is sent to the peer, a counter is incremented and if this
+counter exceeds the tolerance limit, the connection is forcefully closed. The
+counter is reset whenever a ping ack is received from the peer.
+By default, the mechanism for closing the connection based on ping and ping ack
+is disabled.
```

Comment:
We should probably enable it by default, but since *keepalives* are disabled by default, it'll be effectively disabled. I think if people enable keepalives, they will want to have connection close when acks aren't received back.

Reply:
OK, what number do you think we should use as a default? 3?

Reply:
On second thought let's not set a value by default, instead let's mention this option inside the `keepalive` option description.

Reply:
Done.